bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/screens/ExtractionScreen.kt

```diff
@@ -23,8 +21,13 @@ class ExtractionScreen {
             .perform(replaceText(value))
     }
 
-    fun checkTransferSummaryButtonIsClickable(): ExtractionScreen {
-        onView(withId(R.id.transfer_summary)).check(matches(isClickable()))
-        return this
+    fun checkTransferSummaryButtonIsClickable(): Boolean {
+        var isTransferSummaryButtonClickable = false
+        onView(withId(R.id.transfer_summary))  .check { view, noViewFoundException ->
+            if (noViewFoundException == null || view.isClickable()) {
```

Comment:
[Clarification] Same comment as above, about checking exception not thrown AND view is shown.